test/langtools/jdk/javadoc/doclet/testSnippetTag/TestSnippetTag.java

```diff
@@ -2016,6 +2037,17 @@ public void testNegativeHybridTag_Mismatch(Path base) throws Exception {
         checkOutput(Output.OUT, true,
                     """
                     A.java:4: error: contents mismatch""");
+        checkOutput("pkg/A.html", true, """
+                        <details class="invalid-tag">
+                        <summary>invalid @snippet</summary>
+                        <pre>contents mismatch:
+                        ----------------- inline -------------------
+                        Hello, Snippet!
+                        ----------------- external -----------------
+                        Hello, Snippet!...more
+                        </pre>
```

Comment:
FYI, to reference a point made elsewhere, this is one of those cases where we could use `Content` rather than a `String` for the details.  But, no matter for this release.